Lib/typing.py

```diff
@@ -1552,7 +1589,12 @@ def __getitem__(self, params):
             params = (params,)
         msg = "Parameters to generic types must be types."
         params = tuple(_type_check(p, msg) for p in params)
-        _check_generic(self, params, self._nparams)
+        if not self._nparams:
+            raise TypeError(f"{self} is not a generic class")
+        alen = len(params)
```

Comment:
```suggestion
        actual_len = len(params)
```

Let's use the same variable name here as well!